mocks/lists/dataTypesList.js

```diff
@@ -15,13 +16,15 @@ module.exports = {
     ["turtle.com", "turtle"],
     ["donatelo.turtle.com", "turtle"],
     ["rafael.turtle.com", "turtle"],
+    ["leonardo.com","yoshi"]
   ],
 
   SERVICE_TYPE: ["A", "B", "C", "D", "E", "F", "G", "H"],
   UNIT: ["gondor", "mordor", "wallmart", "valhalla"],
   ES_UNIT: ["es1", "es2", "es3", "es4", "es5", "es6"],
   ADS_UNIT: ["ads1", "ads2", "ads3", "ads4", "ads5", "ads6"],
   CITY_UNIT: ["city1", "city2", "city3", "city4", "city5", "city6"],
+  MM_UNIT: ["mm1","mm2","mm3","mm4","mm5","mm6"],
```

Comment:
need to add these units into dataSourcesMap and adjust "createDataSourcesMap" function to map column 'F' to mm